tests/std/tests/P2278R4_views_as_const/test.cpp

```diff
@@ -303,13 +340,16 @@ constexpr bool test_one(Rng&& rng, Expected&& expected) {
     {
         const same_as<iterator_t<R>> auto i = r.begin();
         if (!is_empty) {
+            // (static analyzer doesn't realize that `i == nullptr` implies `is_empty`)
+#pragma warning(suppress : 6011) // Dereferencing NULL pointer 'i'

```

Comment:
No change requested: We avoid `suppress` in `stl/inc` (as it sometimes has issues with templates and/or modules, whereas `push/disable/pop` is reliable) but it's fine in `stl/src` and test code.